### DISCUSSION TITLE
Production ready?

### DISCUSSION THREAD

**Discussion by tuxillo** [loxilb-io/loxilb#859] [General]
Hi,

I'd like to know whether loxilb is production-ready or, if not, in which stages it is.

Thanks

**Comment by TrekkieCoder** [comment]
Hi @tuxillo 

I am one of the maintainer's of loxilb project. But I would still try to give you a very unbiased answer. We try our best to follow a test driven development approach. And most of our features are quite stable at this point. 

However, compared to a proprietary solution, we can't say that all features are 100% production ready. There may be some lesser known/used features which are likely less stable. This is expected of all open-source projects.

If you are serious about considering loxilb, I would suggest making a list of needed features and do lab testing as per your scenarios. And you can work with the community if you face any minor issues.

Further there are also folks who do provide loxilb production support to people in need of the same.

**Comment by tuxillo** [comment]
Thanks @TrekkieCoder for the honest answer!

My question was more towards "is loxilb being used in production" rather than "are all the features production ready", I probably should have been more specific.

Since you already mentioned there are folks providing production support for loxilb I'll assume my original question is answered. I'll leave this discussion opened just in case anybody wants to chime in.
